### ISSUE TITLE
Laravel-menu not working with composer update

### ISSUE THREAD

**Issue by swaplicado** [lavary/laravel-menu#195]
Hi.
Laravel menu was working normally until execute **composer update**, now the view does not recognize the variable that menu "makes":

Middleware:
`\Menu::make('sMenu', function($menu) {`
             ` $menu->add(' ');`
             ` $menu->add('Home', ['route' => 'mms.home']);`
              `$menu->add('About',    'about');`
             ` $menu->add('Services', 'services');`
              `$menu->add('Contact',  'contact');`
              `$menu->add(trans('wms.REPORTS'), 'what-we-do');`
          `});`

View:
`@include(config('laravel-menu.views.bootstrap-items'), array('items' => $sMenu->roots()))`

Error:
Undefined variable: sMenu

Can you help me please?

**Comment by dustingraham** [comment]
Can you ensure that your middleware is executing? It should still create the `$sMenu` global view variable.

I'll try and test this on my end as well.

**Comment by swaplicado** [comment]
Yes, the middleware is executing, It was working before updating composer, that was the only change I made.

I even tried with:
`{!! $sMenu->asUl() !!}`
`{!! Menu::get('sMenu')->asUl() !!}`

but the same thing happens

**Comment by dustingraham** [comment]
What version of php are you using?

**Comment by swaplicado** [comment]
PHP version: 7.17 
Larevel version: 5.1.46 (LTS)

**Comment by dustingraham** [comment]
I just ran some tests, and this is working for me. I used the same declaration you have except I changed Home to just '/' and trans to 'Translated' and it created the menu.

Because you say that `Menu::get('sMenu')->asUl()` is not working, that makes me suspect the `\Menu::make('sMenu', function($menu) {` code is not running.

I understand all that you changed was running composer update, but all signs are pointing to the middleware not being executed.

I did a full test with php 7.1 Laravel 5.5 and then wiped away that test, and did another full test with php 7.1 and Laravel 5.1.* from a fresh install.

I might suggest where you have defined the menu `\Menu::make('sMenu', function($menu) {`

To put on the line immediately above that `die('test');` to confirm that it is in fact executing that code. (Or use a debugger if you have one set up.)
